### ISSUE TITLE
Ingresses don't work out of the box for on-prem Kubernetes deployment.

### ISSUE THREAD

**Issue by paeblits** [Activiti/activiti-cloud-full-chart#563]
I've been trying to deploy the Activiti full example to an on-premise Kubernetes cluster and have found that the Ingresses don't work as expected. I've only seen support for GKE and EKS in the docs, so I wonder if I can get help on this.

I followed instructions in the docs and used Helm with the custom gateway value. After installationI am not able to view any of the services at the URL's that the Helm installation outputs, such as http://gateway-activiti.<mydomain>/modeling. All pods and services are up, and if I change the service to type NodePort it does work. From what I can tell so far the Ingresses seem to be conflicting with each other because if I modify the URL in the modeling Ingress to something different then it does seem to work. 

Any help here is appreciated.

**Comment by unibravoc** [comment]
I have same problem.

**Comment by mteodori** [comment]
you probably haven't installed an ingress implementation on your cluster so no Ingress resource can work

you can try with the Docker Desktop setup and see if you can reproduce your issue: https://github.com/Activiti/activiti-cloud-full-chart#running-on-docker-desktop

**Comment by paeblits** [comment]
We finally diagnosed the problem to be that multiple Ingress objects have the same host rule and were conflicting with each other. Problem is described in more detail here: https://github.com/nginxinc/kubernetes-ingress/issues/76 

Solution described in more detail here: https://github.com/nginxinc/kubernetes-ingress/tree/master/examples/mergeable-ingress-types 

We remedied this by creating a new "master" Ingress with annotation nginx.org/mergeable-ingress-type: master. Then in each Ingress with the same host rule we added the annotation of nginx.org/mergeable-ingress-type: minion.

I'd like to update the documentation to account for this for on-prem deployments.
